rfcs/20190722-tflite-training.md

```diff
@@ -0,0 +1,427 @@
+# On-Device Training with TensorFlow Lite
+
+Status        | Proposed
+:------------ | :---------------------------------------------------------------
+**Author(s)** | Yu-Cheng Ling (ycling@google.com)
+**Sponsor**   | Andrew Selle (aselle@google.com), Jared Duke (jdduke@google.com)
+**Updated**   | 2019-07-22
+
+## Overview & Roadmap
+
+TensorFlow Lite is TensorFlow's recommended solution for on-device machine
+learning. Initially the project focuses on inference, but more and more users
+are asking for on-device training recently.
+
+The doc scopes a multi-quarter effort to get generalized & optimized on-device
+training working with TensorFlow Lite. The project can be broken down into a few
+milestones:
+
```

Comment:
Are there any new APIs being proposed in this doc? I don't see any new concrete APIs so much as ways to extend existing ones to training-- is that an accurate reading?